README.md

```diff
@@ -278,6 +278,36 @@ cat.eatFood("Fish");
 await untilCalled(cat.eatFood(any)); // Completes immediately.
 ```
 
+## Writing a fake
+
+You can also write a simple fake class that implements a real class, by
+extending [Fake]. Fake allows your subclass to satisfy the implementation of
+your real class, without overriding each method with a fake; the Fake class
```

Comment:
how about "without overriding the methods that aren't used in your test"

Reply:
Done. Thanks.